functional_tests/obi/obi_test.go

```diff
@@ -0,0 +1,103 @@
+// Copyright Splunk Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package obi
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/signalfx/splunk-otel-collector-chart/functional_tests/internal"
+)
+
+const (
+	testDir      = "testdata"
+	valuesDir    = "values"
+	manifestsDir = "python"
+)
+
+func Test_OBI_Minimal_Traces(t *testing.T) {
+	if runtime.GOOS != "linux" {
+		t.Skip("OBI functional test runs only on Linux")
+	}
+
```

Comment:
Can you add a teardown call similar to https://github.com/signalfx/splunk-otel-collector-chart/blob/main/functional_tests/token_passthrough/passthrough_test.go#L93 but also add the python app deletion.

Also add a cleanup check at the beginning of the test, in case `SKIP_TEARDOWN` was set .
This might not needed for a one time use cluster, but will be needed when running on a shared cluster in the cloud.
```
	if os.Getenv("TEARDOWN_BEFORE_SETUP") == "true" {
		teardown(t, testKubeConfig)
	}
```

Reply:
Updated. PTAL.